src/CtaButton/CtaButton.stories.js

```diff
@@ -1,181 +1,181 @@
-  href: 'https://stanford.edu',
+  variant: "solid",
+  size: "big",
+  icon: "none",
+  text: "Big Button",
+  href: "https://stanford.edu",
 };
 
 export const Small = CtaButtonTemplate.bind({});
 Small.args = {
-  variant: 'outline',
-  size: 'small',
-  icon: 'none',
-  text: 'Small Button',
-  href: 'https://stanford.edu',
+  variant: "outline",
+  size: "small",
+  icon: "none",
+  text: "Small Button",
+  href: "https://stanford.edu",
 };
 
 export const Action = CtaButtonTemplate.bind({});
 Action.args = {
-  variant: 'solid',
-  size: 'default',
-  icon: 'action',
-  animate: 'right',
-  text: 'Solid with Right Caret',
-  href: 'https://stanford.edu',
+  variant: "solid",
+  size: "default",
+  icon: "action",
+  animate: "right",
+  text: "Solid with Right Caret",
+  href: "https://stanford.edu",
 };
-Action.storyName = 'With Right Caret';
+Action.storyName = "With Right Caret";
 
 export const Download = CtaButtonTemplate.bind({});
 Download.args = {
-  variant: 'solid',
-  size: 'default',
-  icon: 'download',
-  animate: 'down',
-  text: 'Solid Download Button',
-  href: 'https://stanford.edu',
+  variant: "solid",
+  size: "default",
+  icon: "download",
+  animate: "down",
+  text: "Solid Download Button",
+  href: "https://stanford.edu",
 };
-Download.storyName = 'With Download Icon';
+Download.storyName = "With Download Icon";
 
 export const NoAnimate = CtaButtonTemplate.bind({});
 NoAnimate.args = {
-  variant: 'solid',
-  size: 'default',
-  icon: 'email',
-  animate: 'none',
-  text: 'Contact us',
-  href: 'mailto:a@example.com',
+  variant: "solid",
+  size: "default",
+  icon: "email",
+  animate: "none",
+  text: "Contact us",
+  href: "mailto:a@example.com",
 };
-NoAnimate.storyName = 'With Non-animated Icon';
+NoAnimate.storyName = "With Non-animated Icon";
 
 export const SrText = CtaButtonTemplate.bind({});
 SrText.args = {
-  variant: 'solid',
-  size: 'default',
-  icon: 'external',
-  animate: 'top-right',
-  text: 'Learn more',
-  srText: 'about Decanter',
-  href: 'https://stanford.edu',
+  variant: "solid",
+  size: "default",
+  icon: "external",
+  animate: "top-right",
+  text: "Learn more",
+  srText: "about Decanter",
+  href: "https://stanford.edu",
 };
-SrText.storyName = 'With Screen Reader Only Text';
+SrText.storyName = "With Screen Reader Only Text";
 
 export const Custom = CtaButtonTemplate.bind({});
 Custom.args = {
-  variant: 'unset',
-  size: 'unset',
-  icon: 'external',
-  animate: 'top-right',
-  text: 'With Custom Classes',
-  link: 'https://stanford.edu',
+  variant: "unset",
+  size: "unset",
+  icon: "external",
+  animate: "top-right",
+  text: "With Custom Classes",
+  link: "https://stanford.edu",
   className:
-    'su-uppercase su-tracking-widest su-font-semibold su-bg-lagunita-dark hocus:su-bg-plum su-text-white hocus:su-text-white su-p-30',
+    "su-uppercase su-tracking-widest su-font-semibold su-bg-lagunita-dark hocus:su-bg-plum su-text-white hocus:su-text-white su-p-30",
```

Comment:
Looks like the custom padding class `su-p-30` doesn't working anymore but it's not obvious why 🤔 

https://deploy-preview-30--decanter-react.netlify.app/?path=/story/simple-cta-link-button--custom
vs
https://decanter-react.netlify.app/?path=/story/simple-cta-link-button--custom

Reply:
The padding class also stops working for the Button
https://deploy-preview-30--decanter-react.netlify.app/?path=/story/simple-button--custom

I just double checked Decanter 7 - that class should still be there.

Reply:
There was a difference in classes being applied and it had to do with the default value in a switch statement. I made a couple of changes.

![Screen Shot 2021-05-06 at 10 01 12 PM](https://user-images.githubusercontent.com/550602/117400025-a9253180-aeb6-11eb-9241-2d4cc4decaad.png)